ms_endpoint_protection.tf

```diff
@@ -0,0 +1,37 @@
+resource "azurerm_virtual_machine_scale_set_extension" "endpoint_protection" {
+  count = var.install_endpoint_protection == true && var.virtual_machine_type == "vmss" ? 1 : 0
```

Comment:
Can we have this updated to include the OS? E.g. It only supports Windows so irrespective of whether it's enabled it should not install for Linux.

Reply:
Updated as requested/directed.